librz/util/thread.h

```diff
@@ -43,15 +43,11 @@
 #define RZ_TH_LOCK_T pthread_mutex_t
 #define RZ_TH_COND_T pthread_cond_t
 #define RZ_TH_SEM_T  sem_t *
-
+#define RZ_TH_RET_T  void *
 #else
 #error Threading library only supported for pthread and w32
 #endif
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
```

Comment:
why?

Reply:
because is a private header.

Reply:
also it did not have an end-guard. i suspect it was copied from somewhere else.

Reply:
oh right it's in librz/util, ok!